app-modules/authorization/src/Filament/Pages/Auth/Login.php

```diff
@@ -284,4 +304,12 @@ protected function getForms(): array
             ),
         ];
     }
+
+    protected function getAuthenticateFormAction(): Action
+    {
+        return Action::make('authenticate')
+            ->label('Log in')
+            ->submit('authenticate')
+            ->extraAttributes(['class' => 'dark:bg-gray-800 dark_button_border dark:hover:bg-gray-700']);
```

Comment:
Rather than, make a whole new Action, what about returning the `parent::getAuthenticateFormAction()` and add on the extra methods you need.

This way we take advantage of what ever Filament does to that Action in the future while still modifying it to be what we need.